include/nuttx/can.h

```diff
@@ -313,43 +276,85 @@ typedef uint32_t can_err_mask_t;
  * can_dlc: frame payload length in byte (0 .. 8) aka data length code
  *          N.B. the DLC field from ISO 11898-1 Chapter 8.4.2.3 has a 1:1
  *          mapping of the 'data length code' to the real payload length
- * __pad:   padding
- * __res0:  reserved / padding
+ * flags:   higher 8 bits are special frame flag bits
  * __res1:  reserved / padding
  * data:    CAN frame payload (up to 8 byte)
  */
 
-struct can_frame
+begin_packed_struct struct can_frame
```

Comment:
Why do we pack here?
I doesn't guarantee any more that data is word aligned and thus decrease performance?